packages/foreman/nodejs-theforeman-builder/nodejs-theforeman-builder.spec

```diff
@@ -4,287 +4,294 @@
+Source125: https://registry.npmjs.org/leven/-/leven-3.1.0.tgz
+Source126: https://registry.npmjs.org/levenary/-/levenary-1.1.1.tgz
+Source127: https://registry.npmjs.org/locate-path/-/locate-path-2.0.0.tgz
+Source128: https://registry.npmjs.org/lodash/-/lodash-4.17.15.tgz
+Source129: https://registry.npmjs.org/loose-envify/-/loose-envify-1.4.0.tgz
+Source130: https://registry.npmjs.org/minimist/-/minimist-1.2.5.tgz
+Source131: https://registry.npmjs.org/ms/-/ms-2.1.2.tgz
+Source132: https://registry.npmjs.org/node-releases/-/node-releases-1.1.58.tgz
+Source133: https://registry.npmjs.org/object-keys/-/object-keys-1.1.1.tgz
+Source134: https://registry.npmjs.org/object.assign/-/object.assign-4.1.0.tgz
+Source135: https://registry.npmjs.org/p-limit/-/p-limit-1.3.0.tgz
+Source136: https://registry.npmjs.org/p-locate/-/p-locate-2.0.0.tgz
+Source137: https://registry.npmjs.org/p-try/-/p-try-1.0.0.tgz
+Source138: https://registry.npmjs.org/path-exists/-/path-exists-3.0.0.tgz
+Source139: https://registry.npmjs.org/path-parse/-/path-parse-1.0.6.tgz
+Source140: https://registry.npmjs.org/pkg-up/-/pkg-up-2.0.0.tgz
+Source141: https://registry.npmjs.org/private/-/private-0.1.8.tgz
+Source142: https://registry.npmjs.org/regenerate/-/regenerate-1.4.1.tgz
+Source143: https://registry.npmjs.org/regenerate-unicode-properties/-/regenerate-unicode-properties-8.2.0.tgz
+Source144: https://registry.npmjs.org/regenerator-runtime/-/regenerator-runtime-0.13.5.tgz
+Source145: https://registry.npmjs.org/regenerator-transform/-/regenerator-transform-0.14.4.tgz
+Source146: https://registry.npmjs.org/regexpu-core/-/regexpu-core-4.7.0.tgz
+Source147: https://registry.npmjs.org/regjsgen/-/regjsgen-0.5.2.tgz
+Source148: https://registry.npmjs.org/regjsparser/-/regjsparser-0.6.4.tgz
+Source149: https://registry.npmjs.org/resolve/-/resolve-1.17.0.tgz
+Source150: https://registry.npmjs.org/safe-buffer/-/safe-buffer-5.1.2.tgz
+Source151: https://registry.npmjs.org/semver/-/semver-5.7.1.tgz
+Source152: https://registry.npmjs.org/semver/-/semver-7.0.0.tgz
+Source153: https://registry.npmjs.org/source-map/-/source-map-0.5.7.tgz
+Source154: https://registry.npmjs.org/supports-color/-/supports-color-5.5.0.tgz
+Source155: https://registry.npmjs.org/to-fast-properties/-/to-fast-properties-2.0.0.tgz
+Source156: https://registry.npmjs.org/unicode-canonical-property-names-ecmascript/-/unicode-canonical-property-names-ecmascript-1.0.4.tgz
+Source157: https://registry.npmjs.org/unicode-match-property-ecmascript/-/unicode-match-property-ecmascript-1.0.4.tgz
+Source158: https://registry.npmjs.org/unicode-match-property-value-ecmascript/-/unicode-match-property-value-ecmascript-1.2.0.tgz
+Source159: https://registry.npmjs.org/unicode-property-aliases-ecmascript/-/unicode-property-aliases-ecmascript-1.1.0.tgz
+Source160: nodejs-theforeman-builder-%{version}-registry.npmjs.org.tgz
 %if 0%{?scl:1}
 BuildRequires: %{?scl_prefix_nodejs}npm
 %else
 BuildRequires: nodejs-packaging
-BuildRequires: npm
```

Comment:
@sharvit looks like this is why it failed to build

Reply:
probably missing a npm2rpm with this patch: https://github.com/theforeman/npm2rpm/commit/bfcbb9d939a256e426b78a65b88101f2bd5940bf

Reply:
I bring that line back manually, hopefully it's ok